<!-- Logitech Cam -->
<!-- TO SET BEFORE EXPERIMENT -->

<launch>

    
 <include file="$(find nostop_launch_files)/launch/model/cam_model.launch">
  <arg name="camera_name" default="logitech"/>
  <arg name="device" default="/dev/video0"/>
  <arg name="iFOVx" default="45"/>
  <arg name="iFOVy" default="30"/>
 </include> 
  
</launch>
